AntiCovid/app/src/main/java/com/swtug/anticovid/repositories/PreferencesRepo.kt

```diff
@@ -69,4 +69,8 @@ object PreferencesRepo {
             gson.fromJson(json, User::class.java)
         }
     }
+    fun deleteUser(context: Context){
+        getPreferences(context).edit().remove(LOGGED_IN_USER).apply()
+    }
+
```

Comment:
If we logout we also need to remove the vaccination of the user.

Reply:
The Vaccination gets removed with our new commit